.github/workflows/molecule.yml

```diff
@@ -29,10 +29,16 @@ jobs:
       matrix:
         scenario: ${{fromJson(needs.list-scenarios.outputs.matrix)}}
     steps:
-      - uses: actions/checkout@v2
-      - uses: ome/action-ome-ansible-molecule@main
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
```

Comment:
Is the idea to remove the usage of ``action-ome-ansible-molecule`` and replace  it by the code below everywhere? The original action used could be adjusted to cover the new workflow.

Reply:
Discussed briefly with @jmarie - see also the description of https://github.com/IDR/deployment/pull/397. I will push additional commits to these PRs to skip the `action-ome-ansible-molecule` action or the `ome-ansible-molecule` meta-packages and open an issue to discuss a possible deprecation of these repositories.